cpp_tests/test_AddedGenerics.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of TS SAL test suite.
+ *
+ * Developed for the LSST Telescope and Site Systems.
+ * This product includes software developed by the LSST Project
+ * (https://www.lsst.org).
+ * See the COPYRIGHT file at the top-level directory of this distribution
+ * for details of code ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#define CATCH_CONFIG_MAIN
+#include <catch2/catch.hpp>
+
+#include <SAL_Script.h>
+#include <SAL_Test.h>
+
+#include <algorithm>
+
+TEST_CASE("No CSC") {
```

Comment:
In many cases I feel these strings are too short to tell the user what's going on.

Please expand them, or add doc strings when they are too cryptic.

Reply:
Expanded, added description.